mypy/fastparse.py

```diff
@@ -527,7 +527,9 @@ def visit_Try(self, n: ast35.Try) -> TryStmt:
     # Assert(expr test, expr? msg)
     @with_line
     def visit_Assert(self, n: ast35.Assert) -> AssertStmt:
-        return AssertStmt(self.visit(n.test))
+        test = self.visit(n.test)
+        malformed = isinstance(test, TupleExpr) and len(test.items) > 0
```

Comment:
If possible, it'd be better to have this logic in the checker module rather than here -- this module is intended to be primarily concerned with converting the Python AST to the mypy AST.